packages/gatsby-source-shopify/src/type-builders/media-type.ts

```diff
@@ -0,0 +1,151 @@
+export function mediaTypeBuilder(prefix: string): string {
+  return `
+      type ${prefix}ExternalVideo implements Node & ${prefix}Media @dontInfer {
+        alt: String
+        embeddedUrl: String!
+        host: ${prefix}MediaHost!
+        id: ID!
+        mediaContentType: ${prefix}MediaContentType!
+        mediaErrors: [${prefix}MediaError!]!
+        preview: ${prefix}MediaPreviewImage
+        shopifyId: String!
+        status: ${prefix}MediaStatus!
+      }
+
+      interface ${prefix}Media implements Node @dontInfer {
```

Comment:
Are we missing `ShopifyImage` here from `create-resolvers` file?

Reply:
ShopifyImage is not in the media-type file but rather in the common-type as it is used by both the media types (ShopifyMediaImage, Shopify3DModel, ShopifyExternalVideo, ShopifyVideo) as well as by the product directly, collections, etc.